### ISSUE TITLE
How to create a full-width 3 Pane layout with a middle Pane set to `grow`

### ISSUE THREAD

**Issue by bennettrogers** [gfazioli/mantine-split-pane#3]
This project looks great, thank you!

I'm trying to use it to create a 3-pane layout that takes up the full width/height of its container, where the middle pane is set to take all remaining space after setting the left/right pane widths.

Codesandbox [here](https://codesandbox.io/p/sandbox/mantine-split-pane-jk2hhq).

As it is, you can't resize the 3rd pane since the 2nd pane is set to take all remaining space.

Is this layout possible to achieve with this library?

**Comment by gfazioli** [comment]
@bennettrogers hey, I have returned from vacation. Actually, it's a scenario I hadn't considered. Thank you for pointing it out. I will try to find a solution as soon as possible 👍

**Comment by gfazioli** [comment]
@bennettrogers I found a sort of workaround here https://codesandbox.io/p/sandbox/mantine-split-pane-forked-6yvpp7

```ts
import { Split } from "@gfazioli/mantine-split-pane";
import { Paper, Box } from "@mantine/core";
export function Panes() {
  const paperProps = { withBorder: true, w: "100%" };
  const panelLeftInitialWith = 200;
  const panelRightInitialWidth = 250;

  return (
    <Box p={16}>
      <Split w="100%">
        <Split.Pane minWidth={100} initialWidth={panelLeftInitialWith}>
          <Paper {...paperProps}>
            <h1>Pane 1</h1>
          </Paper>
        </Split.Pane>

        <Split.Pane
          initialWidth={`calc(100% - ${panelLeftInitialWith}px - ${panelRightInitialWidth}px )`}
          maxWidth={`calc(100% - ${panelLeftInitialWith}px - ${panelRightInitialWidth}px )`}
        >
          <Paper {...paperProps}>
            <h1>Pane 2</h1>
          </Paper>
        </Split.Pane>

        <Split.Pane minWidth={100} initialWidth={panelRightInitialWidth} grow>
          <Paper {...paperProps}>
            <h1>Pane 3</h1>
          </Paper>
        </Split.Pane>
      </Split>
    </Box>
  );
}
```

I am not satisfied because the `minWidth` settings are not working properly. This requires further investigation.
I will leave this issue open and let you know as soon as possible if I find a way to handle this case.

**Comment by dylburk** [comment]
Any updates on this? I'd love to find a solution where the edges of the outer panes are glued to the edges of their area

**Comment by gfazioli** [comment]
@dylburk did you try the solution above?

**Comment by dylburk** [comment]
@gfazioli Yes, but it seems the minWidth is not working properly as you mentioned above

**Comment by gfazioli** [comment]
@bennettrogers @bennettrogers I found a potential solution. I'm working on it

**Comment by gfazioli** [comment]
@bennettrogers @dylburk I'm working on v2.0! The upcoming version will introduce some breaking changes compared to the current one, but I have managed to enhance the split management and to fix and improve various aspects
